extension.neon

```diff
@@ -1,47 +1,67 @@
 parametersSchema:
-    mahoUseStaticReflection: bool()
-    mahoRootDir: schema(string(), nullable())
     magentoRootPath: schema(string(), nullable())
+    enforceMagicMethodDocBlock: bool()
+    useLocalXml: bool()
 parameters:
-    mahoUseStaticReflection: true
-    mahoRootDir: null
-    magentoRootPath: null
-    bootstrapFiles:
-        - phpstan-bootstrap.php
-    earlyTerminatingMethodCalls:
-        Mage:
-            - throwException
+    magentoRootPath: null # Deprecated
```

Comment:
is this still needed?

Reply:
We don't use it, but if removed then PHPStan will complain about anyone with that config key defined. I.e. if you add this to your phpstan config:
```neon
parameters:
    foo: bar
```
You get the error `Unexpected item 'parameters › foo'.` So I had left it there for people upgrading, but we could remove it too.